### DISCUSSION TITLE
Can't login on GitHub Copilot Chat & Copilot Nightly on insiders build

### DISCUSSION THREAD

**Discussion by Alt-iOS** [community/community#59394] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I accidentally logged out while testing something and now it just won't log back in for the extensions, general level for sync works fine,

This is the output for GitHub authentication:
2023-06-26 21:30:31.700 [info] Reading sessions from keychain...
2023-06-26 21:30:31.700 [info] Getting sessions for all scopes...
2023-06-26 21:30:31.700 [info] Getting sessions for user:email...
2023-06-26 21:30:31.722 [info] Getting sessions for all scopes...
2023-06-26 21:30:31.732 [info] Got stored sessions!
2023-06-26 21:30:31.732 [info] Got 2 verified sessions.
2023-06-26 21:30:31.732 [info] Got 2 sessions for ...
2023-06-26 21:30:31.732 [info] Got 1 sessions for user:email...
2023-06-26 21:30:31.732 [info] Got 2 sessions for ...
2023-06-26 21:30:31.856 [info] Getting sessions for all scopes...
2023-06-26 21:30:31.856 [info] Got 2 sessions for ...
2023-06-26 21:30:31.901 [info] Getting sessions for read:user...
2023-06-26 21:30:31.902 [info] Got 0 sessions for read:user...
2023-06-26 21:30:31.918 [info] Getting sessions for user:email...
2023-06-26 21:30:31.918 [info] Got 1 sessions for user:email...

This is the output for GitHub copilot nightly (which is the same for chat):
[INFO] [auth] [2023-06-27T03:32:29.930Z] Failed to get copilot token due to 401 status. Please sign out and try again.
[ERROR] [default] [2023-06-27T03:32:29.933Z] GitHub Copilot could not connect to server. Extension activation failed: "Your GitHub token is invalid. Please sign out from your GitHub account using VSCode UI and try again."

I add the message it shows
<img width="456" alt="Screenshot 2023-06-26 at 9 34 13 PM" src="https://github.com/community/community/assets/80638470/a2e4317b-8d3a-49f0-847f-65f5d57fe1f8">

I have tried signing out and signing in again, trying normal auth, local server and via 6 digit code, nothing seems to work

Is anyone else having the same problem? Have you guys been able to reproduce it?

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by Alt-iOS** [reply]
Already did so, like 5 times, logging in and out, doesn't change nothing, restarted my Mac, reinstalled VSCode, deleted all cache, nothing changed the messages

**Reply by Alt-iOS** [reply]
Well, you are not going to believe me, it worked, until I closed it and its again asking me the same issues as the begging, is there any permanent fix?

**Reply by davecheney** [reply]
Thank you. Please try one more time. If that does not work, please contact support https://support.github.com/contact

**Comment by abiyevseymur** [comment]
I have same issue. I am using Cursor and sync with VScode. Can't use copilot in Cursor shows same error, but it is works on VScode. Tried to relogin 5-10 times. reinstall extension and cursour IDEA. install again. Nothin helps.

**Comment by AlessandroVol23** [comment]
I have the exact same problem. Logged in and out and reinstalled everything. Any suggestion?

**Comment by realcaldeira** [comment]
install an older version of copilot that works, I currently have v1.168.0

**Comment by Ension-adoin** [comment]
Me too. Any ideas?

**Reply by Ension-adoin** [reply]
Finally, after doing these: uninstall VS Code, clean all the files about VS Code, and reinstall it. Fine now.
